ddapm_test_agent/agent.py

```diff
@@ -212,6 +213,17 @@ def default_value_trace_results_summary():
     }
 
 
+def default_snapshot_server_cassettes_directory():
+    """Return the default directory for snapshot server cassettes.
```

Comment:
instead of hardcoding this to the docker path here can we set a default `ENV SNAPSHOT_SERVER_CASSETTES_DIRECTORY=...` in the Dockerfile?

Reply:
🤦 yes that should work!